### ISSUE TITLE
Plugin stops working after a couple of hours

### ISSUE THREAD

**Issue by sanjitsaluja** [SumoLogic/fluentd-output-sumologic#6]
I'm running 2 docker container my setup. The first container runs our Rails app. The other runs for FluentD collecting logs and sending it to SumoLogic.

When the FluentD container is started, everything works as expected. After a couple of hours FluentD stops sending logs to SumoLogic. At this point of time, FluentD is still collecting logs and dumping them to the `buffer_type:file`. I see files increasing in size but nothing in SumoLogic. Here's my buffer settings:

```
    # buffering
    buffer_type file
    buffer_path /fluentd/buffer/
    flush_interval 10s
    buffer_chunk_limit 16m
    buffer_queue_limit 512
    disable_retry_limit true
    flush_at_shutdown true
    buffer_queue_full_action drop_oldest_chunk
```

How to diagnose or debug this issue? Any pointers appreciated.

**Comment by frankreno** [comment]
@sanjitsaluja : Thanks for the PR.  So in your set up has the PR addressed the above issue?

**Comment by sanjitsaluja** [comment]
Yes

**Comment by frankreno** [comment]
Fixed by pr #7
